### PR TITLE
Improve question function

### LINKED ISSUE #76: question not working with choices
## Expected Behavior

The choices are displayed when asking a question that contains choices.

## Actual Behavior

No choices are displayed: the user can enter any value.

## Steps to Reproduce the Problem

```javascript
#!/usr/bin/env zx

const number = await question("Pick a number: ", {
  choices: ["1", "2"],
});
```

1. Run the above.
2. See that no choices are displayed.

image

## Specifications

- Version: 1.6.0
- Platform: macOS 10.14.6

### PR DESCRIPTION
**Pull request by DavidIsa**
Fixes #76

This PR is adding [inquirer](https://www.npmjs.com/package/inquirer) module for improving the question function and fixing bugs related to choices in the question function. 

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

**Comment by antonmedv** [comment]
> added 28 packages

I think this is way too much.

File: index.mjs [hunk 1]
```diff
@@ -93,23 +93,19 @@ export function cd(path) {
...
-  return answer
+  const defaultType = options && options.choices.length ? 'rawlist' : 'input';
+  const answers = await inquirer.prompt([
+    {
+      prefix: '',
+      choices: [],
+      type: defaultType,
+      ...options,
+      name: 'question',
+      message: query || '',
```
Comment (quantumsheep) [inline index.mjs:104]: Using `??` instead of `||` would be better as it's made for cases like that.

Comment (DavidIsa) [reply index.mjs:104]: Cool, I didn't know about it. I will change it.

**Comment by DavidIsa** [comment]
@antonmedv I thought it would be cool to support all the features that the inquirer lib supports.

Any suggestions on which lib can be used for displaying the choices?

**Comment by antonmedv** [comment]
I think readline is enough. If you want to display choices-> console.log

**Comment by DavidIsa** [comment]
Okay, then I think we can close the pr.

**Pull request closed without merging**